.prettierrc

```diff
@@ -2,5 +2,13 @@
   "bracketSpacing": false,
   "singleQuote": true,
   "trailingComma": "es5",
-  "quoteProps": "preserve"
+  "quoteProps": "preserve",
+  "overrides": [{
+    "files": "OWNERS",
+    "options": {
+      "parser": "json5",
+      "bracketSpacing": true,
+      "trailingComma": "none"
```

Comment:
I recall the trailing comma broke eslint, but since eslint is disabled on this maybe we can re-enable the trailing comma here. What do you think?

Reply:
I kept the config that's currently in use to minimize the changes in this PR. Let's remove the setting and auto-fix all files in a separate PR.

Reply:
SGTM

Reply:
Sent out #25085, which switches `OWNERS` files to the default `json5` format, now that `eslint` is not a consideration.